### PR TITLE
Multiple collections support in the destination

### PR DESCRIPTION
**Pull request by hariso**
### Description

Closes https://github.com/ConduitIO/conduit/issues/1474.

In the absence of the metadata in the Pg source connector, I've used a processor to insert the metadata:
```yaml
---
version: 2.2
pipelines:
  - id: pipeline1
    status: running
    name: pipeline1
    description: 'Generator to PostgreSQL'
    connectors:
      - id: generator-source
        type: source
        plugin: "builtin:generator"
        name: generator-source
        settings:
          recordCount: "10"
          # readTime: "1s"
          format.options: "id:int,name:string"
          format.type: "structured"

      - id: pg-destination
        type: destination
        plugin: "standalone:postgres"
        name: pg-destination
        settings:
          url: "postgresql://meroxauser:meroxapass@localhost/meroxadb?sslmode=disable"
          # table: "employees"
    
    processors:
      - id: "format-lines"
        plugin: "custom.javascript"
        settings:
          script: |
            function getRandomInt(max) {
              return Math.floor(Math.random() * max);
            }

            function getRandomBoolean() {
              return Math.random() < 0.5;

            }

            function process(rec) {
              rec.Key = RawData(JSON.stringify({"id": getRandomInt(1000)}));
              if (getRandomBoolean()) {
                rec.Metadata["opencdc.collection"] = "contractors"
              } else {
                rec.Metadata["opencdc.collection"] = "employees"
              }

              return rec;
            }

```

Tested with a static table name and also the default Go template.

I've renamed the GitHub job `build` to `test` because it more accurately describes what it does. `build` is a required PR check, but I'll handle the rename once the PR has been approved.

### Quick checks:

- [x] I have followed the [Code Guidelines](https://github.com/ConduitIO/conduit/blob/main/docs/code_guidelines.md).
- [x] There is no other [pull request](https://github.com/ConduitIO/conduit-connector-postgres/pulls) for the same update/change.
- [ ] I have written unit tests.
- [x] I have made sure that the PR is of reasonable size and can be easily reviewed.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
-name: build
+name: test
```
Comment (raulb) [inline .github/workflows/test.yml:1]: Thanks for making this change 🙇

**Review by raulb — APPROVED** [review]
LGTM! Once this ships I'll rebase https://github.com/ConduitIO/conduit-connector-postgres/pull/129.

File: destination.go [hunk 2]
```diff
@@ -29,14 +29,16 @@ import (
 const (
 	// TODO same constant is defined in packages longpoll, logrepl and destination
 	//  use same constant everywhere
```
Comment (maha-hajja) [inline destination.go:31]: we should be able to get rid of this comment now, maybe @raulb can you do that in your PR plz?

Comment (raulb) [reply destination.go:31]: @maha-hajja will do!

Comment (raulb) [reply destination.go:31]: https://github.com/ConduitIO/conduit-connector-postgres/commit/5fe1598dcb2fd8613fdcfb6b0ff0bd61d3e2fc02

**Pull request merged**